src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -364,6 +358,23 @@ bool mesh::is_multidomain(const Node &n,
 }
 
 
+//-----------------------------------------------------------------------------
+bool
+mesh::verify(const Node &n,
+             Node &info)
+{
+    return mesh::verify_multidomain(n, info) || mesh::verify_unidomain(n, info);
```

Comment:
One thing I wanted to highlight in this PR was the implementation of this function.  While this function works properly insofar as it returns the correct value, it doesn't always put the right information into the `info` node.  Currently, this function always returns error information from the `blueprint::mesh::verify_unidomain` (i.e. the old `blueprint::mesh::verify` function), which can be incorrect if the user's mesh intends perform a multidomain mesh check.  Going forward, we need to decide what in the structure of the given mesh node classifies it as unidomain or multidomain so that the error output of this function is more helpful for multidomain verifications.